src/padding.rs

```diff
@@ -0,0 +1,53 @@
+use crate::CellOption;
+use papergrid::{Entity, Grid, Settings};
+
+const DEFAULT_PADDING_CHAR: char = ' ';
+
+/// Padding is responsible for a left/right/top/bottom indent of particular and fill one set characters.
+///
+/// ```rust,no_run
+///   # use tabled::{Style, Padding, Row, Table, Modify};
+///   # let data: Vec<&'static str> = Vec::new();
+///     let table = Table::new(&data).with(Modify::new(Row(..1)).with(Padding::new(0, 0, 1, 1).set_char('V', '^', '<', '>')));
+/// ```
+#[derive(Debug)]
+pub struct Padding(usize, char, usize, char, usize, char, usize, char);
```

Comment:
I would rather created a general `struct` here (not tuple based).

Also we could use `Indent` structure from my previous comment.

Reply:
agree👍🏼